br/pkg/task/backup.go

```diff
@@ -718,7 +726,7 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 		}
 	}
 	updateCh = g.StartProgress(ctx, "Checksum", checksumProgress, !cfg.LogProgress)
-	schemasConcurrency := uint(min(backup.DefaultSchemaConcurrency, schemas.Len()))
```

Comment:
```backup.DefaultSchemaConcurrency``` can be removed.

Reply:
it used as initial value for the new config

Reply:
Isn't a default value of 64 a bit too high? Should the default setting be more conservative?

Reply:
two reasons to keep it 64
1. to keep the compatibility.
2. to keep checksum performance.